src/components/DocumentationTopic/TopicsLinkBlock.vue

```diff
@@ -180,8 +180,8 @@ export default {
       if (topic.titleStyle === TitleStyles.title) {
         return topic.ideTitle ? 'span' : 'code';
       }
-      // Framework name links should not be code voice
-      if (topic.role && topic.role === TopicRole.collection) return 'span';
+      // Framework name and property list links and should not be code voice
+      if (topic.role && (topic.role === TopicRole.collection || topic.role === TopicRole.dictionarySymbol)) return 'span';
```

Comment:
I was initially worried that this might not work, because other symbols like REST objects also have a `dictionarySymbol` role, and we _do_ want to use code voice for those. However, after testing a bit, it looks like this is handled by the template, since these topics have `fragments`, which means the `DecoratedTopicTitle` component is used and this code never runs in those instances.

All that to say, seems like this is working even though I didn't expect it to 😅